ledger/error/generator/test/suite/scala/com/daml/error/generator/ErrorCodeDocumentationGeneratorSpec.scala

```diff
@@ -9,22 +9,23 @@ import com.daml.error.utils.testpackage.subpackage.MildErrorsParent.MildErrors
 import com.daml.error.utils.testpackage.subpackage.MildErrorsParent.MildErrors.NotSoSeriousError
 import com.daml.error.utils.testpackage.{DeprecatedError, SeriousError}
 import com.daml.error._
+import com.daml.error.generator.ErrorCodeDocumentationGenerator.DeprecatedItem
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 
-class ErrorCodeDocumentationGeneratorSpec extends AnyFlatSpec with Matchers {
-  private val className = ErrorCodeDocumentationGenerator.getClass.getSimpleName
+import scala.annotation.nowarn
+import scala.reflect.ClassTag
 
-  // Scan errors from the utils test package
-  private val generator = new ErrorCodeDocumentationGenerator(
-    Array("com.daml.error.utils.testpackage")
-  )
+@nowarn("cat=deprecation")
```

Comment:
You could think of moving this annotation to the expression that actually needs to access deprecated items for testing purposes.

Reply:
Ok